mappings/net/minecraft/block/AbstractBlock.mapping

```diff
@@ -91,7 +91,8 @@ CLASS net/minecraft/class_4970 net/minecraft/block/AbstractBlock
 	METHOD method_9535 getRenderingSeed (Lnet/minecraft/class_2680;Lnet/minecraft/class_2338;)J
 		ARG 1 state
 		ARG 2 pos
-	METHOD method_9536 onBlockRemoved (Lnet/minecraft/class_2680;Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;Lnet/minecraft/class_2680;Z)V
+	METHOD method_9536 onStateReplaced (Lnet/minecraft/class_2680;Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;Lnet/minecraft/class_2680;Z)V
+		COMMENT Check if block is removed and perform cleaning operations here. Drop inventories here. Don't forget to call super when you're done.
```

Comment:
Could possibly improve this doc tbh, its not a turtorial.

Reply:
It does say `COMMENT` and not `JAVADOC` ;)